contributing/development/compiling/compiling_with_mono.rst

```diff
@@ -8,119 +8,73 @@ Compiling with Mono
 Requirements
 ------------
 
-- Mono 6.12.0 or greater (generally 64 bit)
-- MSBuild
-- NuGet
-- **On Linux/macOS only:** pkg-config
+- .NET SDK 6.0 or greater (generally 64 bit)
 
-You can use ``mono -V`` to check your Mono version.
-A build error like the one below may indicate your Mono version is too old:
-
-    'mono_runtime_set_pending_exception': identifier not found
-
-You may need to import necessary certificates for NuGet to perform HTTPS
-requests.
-
-The recommended method is to use **curl**'s CA (Certificate Authorities) certificate bundle.
-
-Run the following commands to download and import it. On Windows, you can run it
-from the Mono command line prompt (or the regular prompt if you added Mono's
-``bin`` directory to your ``PATH`` environment variable)::
-
-    # If using PowerShell, replace `curl` with `curl.exe` below.
-    curl -LO https://curl.haxx.se/ca/cacert.pem
-    cert-sync --user cacert.pem
-
-Alternatively, you can use the following command, though it's deprecated and may not work correctly::
-
-    mozroots --import --sync
-
-Environment variables
----------------------
-
-By default, SCons will try to find Mono in the Windows Registry on Windows or
-via ``pkg-config`` on other platforms. You can specify a different installation
-directory by passing the ``mono_prefix`` command-line option to SCons; e.g.
-``scons [...] mono_prefix=%ProgramFiles%/Mono``.
-
-This is the directory that contains the subdirectories ``include`` and ``lib``.
-Note that as usual, paths including spaces must be wrapped in double quotes.
+You can use ``dotnet --version`` to check your .NET version.
```

Comment:
Mentioned `--list-sdks` instead, tho am not sure if it is better now, since that latest version is the only one that really matters anyways.

Reply:
I think it's fine to use `--version`, `--list-sdks` or `--info`, all of those should allow you to check which version(s) of the .NET SDK you have installed, which is the point of this line. I don't think it's too important which one we recommend, I just wanted to be clear about what developers should be looking for and what the output of the command means.

If we tell developers to use `--version` we should be clear that this is the version of the latest .NET SDK you have installed, and that this is likely what you care about, but that you may have other SDKs installed and that's fine too.

If, instead, we want to tell developers to use `--list-sdks` or `--info`, we should clarify what part of the output they should be looking for.

Either way, this is something I'd expect a developer familiar with .NET to already know about, so it's probably not a big deal anyway.

Reply:
I switched to recommending `--info`, since the other options are somewhat broken if no sdk and just a runtime is installed. IMO the output is `--info` is clear enough that that we don't have the clarify things here.

```
root@53c40fc161c2:/# dotnet --version
The command could not be loaded, possibly because:
  * You intended to execute a .NET application:
      The application '--version' does not exist.
  * You intended to execute a .NET SDK command:
      No .NET SDKs were found.

Download a .NET SDK:
https://aka.ms/dotnet-download

Learn about SDK resolution:
https://aka.ms/dotnet/sdk-not-found
root@53c40fc161c2:/# dotnet --list-sdks
root@53c40fc161c2:/# dotnet --info

global.json file:
  Not found

Host:
  Version:      6.0.14
  Architecture: x64
  Commit:       2a90daa2cc

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.NETCore.App 6.0.14 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

Download .NET:
  https://aka.ms/dotnet-download

Learn about .NET Runtimes and SDKs:
  https://aka.ms/dotnet/runtimes-sdk-info
```